apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts

```diff
@@ -26,7 +28,20 @@ export class CreateIntegration {
   ) {}
 
   async execute(command: CreateIntegrationCommand): Promise<IntegrationEntity> {
-    let response: IntegrationEntity;
+    if (command.identifier) {
+      const existingIntegrationWithIdentifier = await this.integrationRepository.findOne({
+        _organizationId: command.organizationId,
+        identifier: command.identifier,
+      });
+
+      if (existingIntegrationWithIdentifier) {
+        throw new BadRequestException('Integration with identifier already exists');
+      }
+    }
```

Comment:
check if the integration with `identifier` already exists for the organization, if yes throw an error

Reply:
wdyt about throwing ConflictException here?